content/en/docs/concepts/workloads/pods/disruptions.md

```diff
@@ -233,15 +233,9 @@ can happen, according to:
 
 ## Pod disruption conditions {#pod-disruption-conditions}
 
-{{< feature-state for_k8s_version="v1.26" state="beta" >}}
+{{< feature-state feature_gate_name="PodDisruptionConditions" >}}
 
-{{< note >}}
-In order to use this behavior, you must have the `PodDisruptionConditions`
-[feature gate](/docs/reference/command-line-tools-reference/feature-gates/)
-enabled in your cluster.
-{{< /note >}}
-
-When enabled, a dedicated Pod `DisruptionTarget` [condition](/docs/concepts/workloads/pods/pod-lifecycle/#pod-conditions) is added to indicate
+A dedicated Pod `DisruptionTarget` [condition](/docs/concepts/workloads/pods/pod-lifecycle/#pod-conditions) is added to indicate
```

Comment:
wrap lines at 80 characters (outside of links)

Reply:
Ok, I broke the long line after the link. 

However, I didn't reshuffle the whole paragraph as this would decrease readability of the diff, and more likely result in unnecessary conflicts. There are many long lines in the text, so I think we could have a dedicated issue / pr to clean up formatting holistically (but even then 120 chars would be my preference :) )